### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by jcromerohdz**

### CODE REVIEW

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review by: Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Nice work team :+1: 
### But there are changes to implement
- [x] Validate your html file. [Html Validator](https://validator.w3.org) Ensure there are no errors.
- [x] Usage of semantic html, you forgot to add ```<main>``` *tag* to specify the main content of your html document.
- [x] In your navigation section, the ```<nav>``` *tag* should use a ```<div>``` *tag* to position elements.  Inspect the source page and use it a reference.
 - [x] Add more contents to your README
     * Add the link to Odin project for vistors to your github account to get more information
     * You can write a terse of what Tear-down is, because it is not your usual website :smiley: 
     * [**optional**] Add any detail you think would inform those that are going through your project.
### Implement these changes and submit another form for code review :muscle: :fire: 

#### Cheers Henry :smiley:

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+
+## Description
+This is a design teardown practice with css grid.
+In this project we need to create a heap map to represent the containers on the different sections of the smashing magazine web page.
+
+## URL
+[live_preview](http://htmlpreview.github.io/?https://github.com/jcromerohdz/design_teardown/blob/layout/index.html)
+
+## URL
+[To more info ](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown
```
Comment (samgaco) [inline README.md:11]: You haven't closed the last parenthesis

**Review by samgaco — APPROVED** [review]
#### Hi @jcromerohdz and @lhscar56 :high_brightness: 

#### All the requested changes have been implemented therefore you are approved :100: Congratulations :1st_place_medal: 

**[OPTIONAL]** As a suggestion I would tell you to improve the readme file, you still can add your contact details, more information, what have you used to achieve it, what is the project about, etc. You can find inspiration in readme templates like [this one](https://github.com/othneildrew/Best-README-Template) or free courses like [this one.](https://www.udacity.com/course/writing-readmes--ud777)


--------------

* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**